developer.md

```diff
@@ -6,10 +6,10 @@ There are many ways you can setup you local environment, this is just a basic qu
 
 ### Pre-requisites
 To follow this section it is assumed you have:
-* Golang >= 1.16 installed [https://golang.org/doc/install](https://golang.org/doc/install)
+* Golang >= 1.18 installed [https://golang.org/doc/install](https://golang.org/doc/install)
```

Comment:
Hey, these are good updates, but we do something strange for some of these files. They are copied over from docs/*.md on commit to the master branch. So for the changes to developer.md, quickstart.md, and libraries.md, lets put them in the versions here https://github.com/G-Research/armada/tree/master/docs and automation will pull them over. (So you'll need a new PR, off master branch and targeted at master branch).